src/Components/LoginForm.tsx

```diff
@@ -0,0 +1,233 @@
+import React, {
+  FormEvent,
+  useContext,
+  useMemo,
+  useRef,
+  useState,
+} from 'react';
+import classNames from 'classnames';
+import { AppContext } from '../AppContext';
+import { findUser, addUsers } from '../api/todos';
+import { Notification } from './Notification';
+
+const addPrevUser = (prevEmail: string) => {
+  const getPrevUsers: string[] | 0 = JSON
+    .parse(localStorage.getItem('prevusers') || '0');
+
+  if (getPrevUsers !== 0) {
+    if (getPrevUsers.length === 5) {
```

Comment:
If I look on the variable name `getPrevUsers` I think that it's a function. But how can the function have a length property? So, rename the variable, please